src/Commands/BadasoLMSSetup.php

```diff
@@ -35,23 +38,88 @@ public function __construct() {
      *
      * @return void
      */
-    public function handle(): void {
+    public function handle() {
         $this->addingBadasoEnv();
         $this->publishBadasoProvider();
-        $this->addPostTablesToHiddenTables();
+        $this->addLMSTablesToHiddenTables();
         $this->linkStorage();
-        $this->generateSwagger();
     }
 
-    protected function generateSwagger() {}
+    protected function publishBadasoProvider() {
+        Artisan::call('vendor:publish', ['--tag' => 'BadasoLMSModule']);
 
-    protected function publishBadasoProvider() {}
+        $this->info('Badaso LMS provider published');
+    }
+
+    protected function linkStorage() {
+        Artisan::call('storage:link');
+    }
+
+    protected function envListUpload() {
+        return [
+            'MIX_LMS_URL_PREFIX' => '/lms',
+            'MIX_FRONTEND_URL' => 'http://localhost:8000',
+        ];
+    }
+
+    protected function addingBadasoEnv() {
```

Comment:
prefer to use verb1 for function name, e.g. `addBadasoEnv`

Reply:
fixed